__tests__/app.test.js

```diff
@@ -48,3 +48,33 @@ describe("GET /api/topics", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  test("200: returns a single article with the required properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.article;
+        expect(articles).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          body: expect.any(String),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+
+  test("GET:400 sends an appropriate status and error message when given an invalid id", () => {
+    return request(app)
+      .get("/api/articles/article")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body).toEqual({ message: "Bad request" });
+      });
+  });
+});
```

Comment:
You can test for another status and error message rather than just a bad request, if a user makes a request but then no info is being returned what feedback would a user expect? this is functionality and therefore should be tested for also.